Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -889,7 +889,7 @@ public extension ClusterSystem {
         let recipient = _ActorRef<InvocationMessage>(.remote(.init(shell: clusterShell, address: actor.id._asRemote, system: self)))
 
         let arguments = invocation.arguments
-        let ask: AskResponse<Res> = recipient.ask(timeout: .seconds(5)) { replyTo in
+        let ask: AskResponse<Res> = recipient.ask(timeout: RemoteCall.timeout ?? self.settings.callTimeout) { replyTo in
```

Comment:
There is already `self.settings.callTimeout`, which based on the description is for this purpose. I can rename this to `remoteCallTimeout` if we prefer that.

Reply:
The rename sounds good -- `remoteCallTimeout` or maybe `defaultRemoteCallTimeout` even.

Sounds good 👍 

Is the value there also pretty large, like 5 sec?

Reply:
`defaultRemoteCallTimeout` it is